### ISSUE TITLE
Website

### ISSUE THREAD

**Issue by cjerzak** [cjerzak/fastrerandomize-software#11]
Questions to ponder: (1) where to host package website; (2) URL (maybe just fastrerandomize.org or https://fastrerandomize-labs.github.io/).  https://grf-labs.github.io/ has a great web interface we can take inspiration from.

**Comment by WarrenZhu050413** [comment]
Yes. https://pkgdown.r-lib.org/ seems to be what `grf` uses. I think it is great

**Comment by cjerzak** [comment]
We're thinking this: fastrerandomize.github.io

**Comment by WarrenZhu050413** [comment]
https://fanwangecon.github.io/R4Econ/support/development/fs_packaging.pdf

**Comment by WarrenZhu050413** [comment]
Decided to use package down.

**Comment by cjerzak** [comment]
Another website to take inspiration from: https://mc-stan.org/

**Comment by cjerzak** [comment]
Update: Per group discussion, we are moving forward with github.io + https://pkgdown.r-lib.org/
